docs/validators/sigstore_cosign.md

```diff
@@ -0,0 +1,144 @@
+# SigStore / Cosign
```

Comment:
If you want to capitalize this S of Store, do it for the others below and all the other files as well

Reply:
decapped